lib/components/Taskfile.yml

```diff
@@ -0,0 +1,46 @@
+version: "3"
+
+tasks:
+  init:
+    desc: Initialize a new Masterpoint's component directory and fetch its configuration and sources.
+    requires:
+      vars:
+        - COMPONENT_NAME
+        - COMPONENT_VERSION
+    silent: true
+    cmds:
+      - echo '👋 Initializing component {{.COMPONENT_NAME}}...'
+      - mkdir -p ./components/{{.COMPONENT_NAME}}
+      - echo 'Fetching component config file...'
+      - |
+        curl -L https://raw.githubusercontent.com/masterpointio/terraform-components/$COMPONENT_VERSION/templates/component.yaml \
+        | sed -e 's|COMPONENT_NAME|{{.COMPONENT_NAME}}|g' -e 's|COMPONENT_VERSION|{{.COMPONENT_VERSION}}|g' \
+        > ./components/{{.COMPONENT_NAME}}/component.yaml
+      - task: update
+
+  update:
+    desc: Update an existing component from its source based on the config information.
+    requires:
+      vars:
+        - COMPONENT_NAME
+    vars:
+      COMPONENT_PATH: "components/{{.COMPONENT_NAME}}"
+      COMPONENT_CONFIG: "{{.COMPONENT_PATH}}/component.yaml"
+      COMPONENT_SOURCE_URI:
+        sh: yq eval '.component.source.uri' {{.COMPONENT_CONFIG}}
```

Comment:
We should start moving towards [`dasel`](https://daseldocs.tomwright.me/examples/basics). Not necessary as part of this PR, but just calling it out.